### DISCUSSION TITLE
Backlog

### DISCUSSION THREAD

**Discussion by claraquintela** [582-41E-MA/projet-web-2-CollRen#91] [Q&A]
Salut les gars @CollRen et @NabilDjerroud 

Avez vous changer le backlog sur excel? J'ai aperçu qu'il n'y a pas les taches du sprint 2. Marcos veut bien regardé ça. Ou c'est moi qui ne sait pas lire? 

Desolée de vous perturber, mais sans ça, nous avons pas la remise complète....

**Comment by claraquintela** [comment]
Bonjour, les gars
Aujourd'hui c'est notre deadline pour la remise

Avez vous eu du temps de regarder le problème sur le back log?

Merci :) 

@CollRen @NabilDjerroud

**Reply by NabilDjerroud** [reply]
Bonjour Clara, oui, c'est fait, je viens de m'apercevoir, en mettant un filtre pour le sprint 2 qu'il y avait une tache qui n'était à jour, c'est fait maintenant, merci Clara

**Reply by claraquintela** [reply]
Le problème c'est que je ne vois pas les sprint 2. Je crois que @CollRen a mis dans l'archive, mais ça a changé les infos que on avait écrit

**Reply by claraquintela** [reply]
Laissez tombé mon commentaire avant. C'est tout bon. Desolée de vous dérangé, specialement @CollRen  :)
